recipe/bld.bat

```diff
@@ -4,19 +4,29 @@ if errorlevel 1 exit 1
 cd build
 if errorlevel 1 exit 1
 
+set BUILDCONF=Release
+
+set "CMAKE_COMPILER_PATH=%VSINSTALLDIR%\..\..\VC\bin\amd64"
```

Comment:
`cmake` doesn't like back slashes... I wonder if this can be changed to something like:
```
set "CMAKE_COMPILER_PATH=%VSINSTALLDIR:\=/%/../../VC/bin/amd64"
```
So it is all forward slashes....